### PR TITLE
added NOP keygen script & set-node-key cmd

### PR DESCRIPTION
**Pull request by KuphJr**

### CODE REVIEW

File: tasks/Functions-admin/setNodeKey.js
Comment (pinebit) [inline tasks/Functions-admin/setNodeKey.js:28]: Wrong message

File: tasks/Functions-admin/setNodeKey.js
Comment (pinebit) [inline tasks/Functions-admin/setNodeKey.js:36]: Wrong message

**Review by pinebit — CHANGES_REQUESTED** [review]

File: tasks/Functions-admin/setNodeKey.js
Comment (bolekk) [inline tasks/Functions-admin/setNodeKey.js:12]: We shouldn't be copy-pasting this block in every command. At least make it a shared functions.

Comment (KuphJr) [reply tasks/Functions-admin/setNodeKey.js:12]: Is this change going to block this PR or can we merge it for now?  Changing this is going to require changing every task file in the repo.

Comment (bolekk) [reply tasks/Functions-admin/setNodeKey.js:12]: I haven't "requested changes" so once @pinebit approves you're good to go.

File: tasks/Functions-admin/setNodeKey.js
Comment (pinebit) [inline tasks/Functions-admin/setNodeKey.js:16]: I would wish this param to be configured globally. Please create a ticket for this.

**Review by pinebit — APPROVED** [review]

**Pull request merged**
